force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -93,23 +150,50 @@ public with sharing class DummyJSONCallout {
      */
 	
 	public static void postCreateDummyJSONUser(String contactId) {
-		// Create HTTP request to send.
 
-		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
+		Contact cont = [SELECT Id, FirstName, LastName, Email, Phone FROM Contact WHERE Id = :contactId LIMIT 1];
 
+		// Create HTTP request to send.
+		HttpRequest request = new HttpRequest();
+		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
+		request.setEndpoint('https://dummyjson.com/users/add');
 		// Set the HTTP method to POST.
+		request.setMethod('POST');				
+		request.setHeader('Content-Type', 'application/json');
+		// Set the body using generateDummyJsonUserPayload method.					
+		request.setBody('{"firstName": "Tester"}');
 
-		// Set the body using generateDummyJsonUserPayload method.
 
 		// Send the HTTP request and get the response.
+		Http http = new Http();
+		HttpResponse response = http.send(request);
+
+		//Log responses for debugging
+		System.debug(response.getBody());
+		System.debug(response.getStatusCode());
+		System.debug(response.getStatus());
+
+		// If the HTTP response code is successful, 
+		if (response.getStatusCode() >= 200 || response.getStatusCode() <= 299) {
```

Comment:
```suggestion
		if (response.getStatusCode() >= 200 && response.getStatusCode() <= 299) {
```
This should be AND not OR. You want to check the response is between 200 and 299, that indicates a success. Other status codes would mean it failed for some reason.